internal/brokers/auth/consts.go

```diff
@@ -0,0 +1,25 @@
+// Package auth contains the authentication related code.
+package auth
+
+const (
+	// Granted is the response when the authentication is granted.
+	Granted = "granted"
+	// Denied is the response when the authentication is denied.
+	Denied = "denied"
+	// Cancelled is the response when the authentication is cancelled.
+	Cancelled = "cancelled"
+	// Retry is the response when the authentication needs to be retried (another chance).
+	Retry = "retry"
+	// Next is the response when another MFA (including changing password) authentication is necessary.
+	Next = "next"
```

Comment:
How about making these more descriptive by appending `Reply` or `Response` to each?

```suggestion
	NextResponse = "next"
```

Reply:
Not sure about... In case I think it would be better to use it as a prefix so that we've a kind of enum-style but, since we already had around `auth.Next` mentioned in the code it looked consistent with that.

Let's see if @denisonbarbosa has opinions on this too.

Reply:
> In case I think it would be better to use it as a prefix so that we've a kind of enum-style

I think it makes variable names easier to read when using the term that describes what the variable actually is (a response in this case) as the last word in the name, not the first.

> but, since we already had around auth.Next mentioned in the code it looked consistent with that.

I really don't think `auth.Next` makes it clear that this variable represents an authentication response.

Reply:
I was mostly mentioning that because it makes easier to understand what a variable belongs to, and to improve completion so that you know that something in a given category has always the same prefix.